### ISSUE TITLE
pillow: outdated import

### ISSUE THREAD

**Issue by blshkv** [inguma/bokken#13]
```
--- bokken.orig/ui/graph.py  2015-06-06 18:38:36.000000000 +0800
+++ bokken/ui/graph.py       2015-06-06 20:53:09.336555131 +0800
@@ -19,7 +19,7 @@
 
 import gtk
 
-import Image
+from PIL import Image
 import os, tempfile
 from subprocess import *
```

**Comment by Jo-Con-El** [comment]
Oh, I didn't know that they moved to the second form!  Thanks a lot!

**Comment by Jo-Con-El** [comment]
@hteso fixed this in 6fcb2f8 almost a month ago.

**Comment by blshkv** [comment]
ok, I was waiting for the 1.7 STABLE release just to realise that it is really not.

Could you start tagging releases more often please? For example, it might time for 1.7.1 bug fix release.

**Comment by Jo-Con-El** [comment]
1.8 is coming up next Monday! :smile: But I can tag this 1.7-rc if you like it...

**Comment by blshkv** [comment]
as long as it is not an another 3 years it's fine with me ;-) Cheers.

**Comment by Jo-Con-El** [comment]
Beginning of next week.  I promise.

**Comment by Jo-Con-El** [comment]
Aaaand we released Bokken 1.8. :+1: Enjoy.
